src/app/friends/friends.module.ts

```diff
@@ -4,14 +4,20 @@ import { AddEditFriendsComponent } from './add-edit-friends/add-edit-friends.com
 import { FriendListComponent } from './friend-list/friend-list.component';
 
 
+import { RouterModule,Routes } from '@angular/router';
 
+const friendsRoutes : Routes = [
+  {path:'fList', component :FriendListComponent},
+  {path:'fAddEdit', component :AddEditFriendsComponent}
+]
```

Comment:
Use better naming conventions for all route path as I mentioned in other comments